opentracing-spring-jaeger-starter/src/main/java/io/opentracing/contrib/java/spring/jaeger/starter/JaegerConfigurationProperties.java

```diff
@@ -74,6 +89,55 @@ public void setExpandExceptionLogs(boolean expandExceptionLogs) {
     this.expandExceptionLogs = expandExceptionLogs;
   }
 
+  public Map<String, String> getTags() {
+    return tags;
+  }
+
+  public void setTags(Map<String, String> tags) {
+    this.tags = tags;
+  }
+
+  public Map<String, String> determineTags() {
+    if (!includeJaegerEnvTags) {
```

Comment:
Am I correct that env tags:
*  `includeJaegerEnvTags=true` - will be included
* `includeJaegerEnvTags=false` - will not be included

Reply:
That is correct

Reply:
Would you prefer another name?